conanfile.py

```diff
@@ -19,19 +19,19 @@
 
 try:
     from conans import __version__ as conan_version
-    # check conan version
-    if conan_version < "1.40.1":
+    if int(conan_version[0]) >= 2:
         print(failColor + "conan version " + conan_version + " is not compatible with Basilisk.")
-        print("use version 1.40.1+ to work with the conan repo changes from 2021." + endColor)
+        print("use version 1.40.1 to 1.xx.0 to work with the conan repo changes." + endColor)
```

Comment:
Do you mean "1.40.1+"?

Reply:
I didn't think 1.40.1+ was clear enough, as we can't allow versions 2.xx.x.  Thus, I changed the language to be
1.40.1 to 1.xx to indicated we must have version 1, but higher than 1.40.1